lib/dry/events/bus.rb

```diff
@@ -76,6 +76,11 @@ def subscribe(event_id, query, &block)
       def subscribed?(listener)
         listeners.values.any? { |value| value.any? { |(block, _)| block.equal?(listener) } }
       end
+
+      # @api private
+      def register_event?(event_id)
+        events.keys.any? { |value| value == event_id }
```

Comment:
This can be simplified as `events.key?(event_id)`

Reply:
@solnic do we store nils as values? If not it could be simplified to !events[event_id].nil? as it is faster

Reply:
@mensfeld if you do `register_event(nil)` then it'll be stored, but we shouldn't allow that 😅

Reply:
@solnic so my point is valid ;) @GustavoCaso maybe we should add a dry-validation to validate the format of the event ;)